### ISSUE TITLE
Compression issues & artifacts with "Awkward Tower" wallpaper

### ISSUE THREAD

**Issue by fossfreedom** [UbuntuBudgie/budgie-wallpapers#2]
@HEXcube please can you revisit https://github.com/UbuntuBudgie/budgie-wallpapers/blob/master/awkward_tower_by_michael_gaida.jpg

I've just applied that as my background and because of all the straightlines I can see lots and lots of rasterized lines

![Awkward Tower](https://cloud.githubusercontent.com/assets/996240/23133803/51a0eac2-f78a-11e6-8fee-8362d54b0c27.png)

I would strongly recommend we revisit all the wallpapers that has been optimized just incase they suffer the same issues

**Comment by RohanVilloth** [comment]
Looks like it's an issue with the lossy compression I had to apply on some wallpapers. :confused: Most other wallpapers were generated by lossless compression. I'll double check all the wallpapers and compress the affected ones again from source. :+1:

**Comment by RohanVilloth** [comment]
I checked the original "Awkward Tower" foto from [flicker page](https://www.flickr.com/photos/145756788@N06/31916128870/sizes/l), and it had artifacts too! :open_mouth:

[See Image Link](https://c1.staticflickr.com/1/297/31916128870_57470ba219_o.jpg) (Not directly loading the pic here coz it's size is huge!)

Apart from the inherent quality loss in the original image, I couldn't find any significant new artifacts in the processed image. What do you think @fossfreedom ? :confused:

**Comment by fossfreedom** [comment]
damn. you are right!

ok - we cant have that picture.  Lets change it for either https://www.flickr.com/photos/137559118@N08/31875846380/in/pool-ubuntubudgie-1704/

or 

https://www.flickr.com/photos/137559118@N08/31947490290/in/pool-ubuntubudgie-1704/

pick whichever.

**Comment by RohanVilloth** [comment]
The second one, definitely! :smiley:

![Walk Day](https://farm1.staticflickr.com/763/31947490290_025727aca2_z_d.jpg)

[World Photo Walk Day](https://www.flickr.com/photos/137559118@N08/31947490290/in/pool-ubuntubudgie-1704/) by Mr Drizz

**Comment by RohanVilloth** [comment]
But, didn't we select the previous image already for contest and package it? I thought it already hit the Debian and Ubuntu repos :confused:

**Comment by fossfreedom** [comment]
@HEXcube yeah - dont worry - this is really a bug  - the contest picture is something we cannot use - so we'll upload a new package.  Do your magic optimisation thingy - delete the old wallpaper and we can update the various files with the new picture by Mr Drizz

**Comment by RohanVilloth** [comment]
K then. I'll get the new image and do the "magic optimization thingy" :smile:

**Comment by RohanVilloth** [comment]
K.. David, replaced the affected wallpaper with the new one :+1:

However, I noticed something strange while optimizing the new image: scaling down with this command

```sh
mogrify -resize 3840x <image.jpg>
```

increased the image size! :open_mouth: So, I just kept the original size as it was comparatively small (around 750KB) for such a big image and just ran these ones:

```sh
jhead -autorot -de -di -du -c <image.jpg>
/opt/mozjpeg/bin/jpegtran -optimize -progressive -outfile 'Output.jpg' 'Input.jpg'
```

The final result was around 650KB. However, I need to look into why the scaling increased image. Maybe the original image was already compressed with a better algorithm or process. Anyway, it remains a mystery for now :laughing:
